src/test/java/edu/ithaca/dragon/par/io/CohortRecordTest.java

```diff
@@ -152,27 +166,110 @@ public void cohortRecordsToFromJsonTest() throws IOException {
+
+        // test for file with different order/ followup inclusions
+        JsonIoUtil jsonIoUtil = new JsonIoUtil(new JsonIoHelperDefault());
+        List<QuestionOrderedInfo> toPassToOTG2 = CohortRecord.createQuestionOrderedInfoList("src/test/resources/author/orderedQuestionInfo/OrderedQuestionInfoListTest1.json", questionPool);
+        OrderedTaskGenerator testOTG2 = new OrderedTaskGenerator(questionPool, toPassToOTG2);
+        assertNotNull(testOTG2.getQuestionOrderedInfoList());
+        List<QuestionOrderedInfo> checkCustom = jsonIoUtil.listFromFile("src/test/resources/author/orderedQuestionInfo/OrderedQuestionInfoListTest1.json", QuestionOrderedInfo.class);
+        List<QuestionOrderedInfo> fromOTG2 = testOTG2.getQuestionOrderedInfoList();
+        assertEquals(checkCustom.size(), fromOTG2.size());
+        for (int i = 0; i < checkCustom.size(); i++){
+            assertEquals(checkCustom.get(i).getQuestionID(), fromOTG2.get(i).getQuestionID());
+            assertEquals(checkCustom.get(i).isIncludesFollowup(), fromOTG2.get(i).isIncludesFollowup());
+        }
+
+        // test for non existent file; creates default with true followup and prints warning
+        List<QuestionOrderedInfo> toPassToOTG3 = CohortRecord.createQuestionOrderedInfoList("src/test/resources/author/orderedQuestionInfo/fileDoesNotExist.json", questionPool);
+        OrderedTaskGenerator testOTG3 = new OrderedTaskGenerator(questionPool, toPassToOTG3);
+        assertNotNull(testOTG3.getQuestionOrderedInfoList());
+        List<QuestionOrderedInfo> checkDoNotExist = OrderedTaskGenerator.createDefaultQuestionOrderedInfoList(questionPool, true);
+        List<QuestionOrderedInfo> fromOTG3 = testOTG3.getQuestionOrderedInfoList();
+        assertEquals(checkDoNotExist.size(), fromOTG3.size());
+        for(int i = 0; i < checkDoNotExist.size(); i++){
+            assertEquals(checkDoNotExist.get(i).getQuestionID(), fromOTG3.get(i).getQuestionID());
+            assertEquals(checkDoNotExist.get(i).isIncludesFollowup(), fromOTG3.get(i).isIncludesFollowup());
+        }
+    }
+
+    @Test
+    public void overwriteCohortDatastoreFileTest() throws IOException {
+        JsonIoUtil jsonIoUtil = new JsonIoUtil(new JsonIoHelperDefault());
+        //read in 2 cohort datastores- 1 not altered for comparison, 1 to alter
+        List<CohortRecord> testCohortRecords = jsonIoUtil.listFromFile("src/test/resources/author/defaultCohortDatastore.json", CohortRecord.class);
+        JSONCohortDatastore toReference = CohortRecord.makeCohortDatastoreFromCohortRecords(testCohortRecords, "src/test/resources/author/defaultCohortDatastore.json");
+        JSONCohortDatastore toTest = CohortRecord.makeCohortDatastoreFromCohortRecords(testCohortRecords, "src/test/resources/author/defaultCohortDatastore.json");
+
+        //ask for TaskGenerator of studentID that does not exist
+        TaskGenerator taskGeneratorRequested = toTest.getTaskGeneratorFromStudentID("taskGeneratorStudent");
```

Comment:
would proabably be good to check that it wasn't there before calling to ensure you're testing the right thing.